common/source/docs/common-fettec-onewire.rst

```diff
@@ -0,0 +1,146 @@
+===================
+FETtec OneWire ESCs
+===================
+
+.. note::
+
+   This feature is only available on github master branch and on ArduPilot-4.1 (and higher)
+
+.. image:: ../../../images/FETtec_4in1_ESC.jpg
+
+OneWire is an `ESC bi-directional communication protocol<https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_FETtec_OneWire>`_ from FETtec.
```

Comment:
this is a malformed link..it needs a space in front of the first brace on url..and it should not point to the code, but rather an explanation of the protocol (https://blog.seidel-philipp.de/what-is-fettec-onewire/), or just remove the link entirely

Reply:
It does not point to code, it points to a README.md file AND the source code! All in one! :)